public/assets/lib/cluster/MarkerCluster.css

```diff
@@ -0,0 +1,6 @@
+.leaflet-cluster-anim .leaflet-marker-icon, .leaflet-cluster-anim .leaflet-marker-shadow {
```

Comment:
Tu as importé la lib Leaflet ? 
On peut peut être l'installer en tant que dépendance npm pour garder les mises à jour et éviter de versionner du code qui n'est pas à nous 

npm install --save leaflet ?

Reply:
Bon j'ai testé mais j'ai "ReferenceError: exports is not defined" vraisemblablement venant de la version d'ES5/6, je te laisserai regarder ça ultérieurement